### PR TITLE
added test and also ran with pytest. tested both function in main.py …

### PR DESCRIPTION
**Pull request by Michellebuchiokonicha**
…and database.py

### CODE REVIEW

**Comment by Michellebuchiokonicha** [comment]
test created in todo-app-python

File: todo-backend/todo-app-python/test_todos.db
Comment (miccheng) [inline todo-backend/todo-app-python/test_todos.db:1]: Should probably add this file to .gitignore

File: todo-backend/todo-app-python/venv/bin/Activate.ps1
Comment (miccheng) [inline todo-backend/todo-app-python/venv/bin/Activate.ps1:1]: I think u checked in the venv folder. We don’t actually need this when running in the Devcontainer. Can remove?

Should probably add to .gitignore as well.

**Review by miccheng — CHANGES_REQUESTED** [review]
Some changes needed.

**Review by Michellebuchiokonicha — COMMENTED** [review]
I added test_todos.db and venv to the .gitignore

**Comment by miccheng** [comment]
LGTM

**Pull request merged**